components/resume/company/company.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react'
+import Position from './positions';
+import Skills from './skills';
+
+interface Position {
```

Comment:
This is getting repeated, so may make sense to break out into a seperate/common type file

Reply:
Added this as a tracker item to https://github.com/jarekb84/me/issues/1